tests/unit/visualise/test_generate_map_based_visualisations.py

```diff
@@ -18,6 +19,12 @@ def AURN_filepath(sampledir):
     return AURN_filepath
 
 
+@pytest.fixture()
+def AURN_savepath(tmp_output_path):
```

Comment:
Whilst we're tweaking this, could you change the names of these `AURN_*` functions to be lower case, like this
```suggestion
def aurn_savepath(tmp_output_path):
```
As that's the convention for function names (Names starting with uppercase are the convention for classes)